src/weather_gen.h

```diff
@@ -12,29 +12,29 @@ class JsonObject;
 enum weather_type : int;
 
 struct w_point {
-    double temperature;
-    double humidity;
-    double pressure;
-    double windpower;
+    double temperature = 0.0f;
```

Comment:
```suggestion
    double temperature = 0;
```
And so on for the below.